app/utils/xpath.py

```
@@ -168,6 +181,20 @@ class Past:
 
 class Competitions:
     class Profile:
+        LEAGUE_COUNTRY_NAME = "//div[@class='data-header__club-info']//following::span[1]/a/text()[1]"
+        LEAGUE_COUNTRY_ID = "//div[@class='data-header__club-info']//following::span[1]/a//@href"
+        LEAGUE_TIER = "//div[@class='data-header__club-info']//following::span[2]//span[2]/text()"
+        REIGNING_CHAMPION = "//div[@class='data-header__club-info']//following::span[3]//text()//following::span[1]/a/text()[1]"
+        RECORD_HOLDING_CHAMPION = "//div[@class='data-header__club-info']//following::span[4]//text()//following::a/text()[1]"
+        UEFA_COEFFICIENT_POSITION = "//div[@class='data-header__club-info']//following::span[5]//text()//following::a/text()[1]"
+        UEFA_COEFFICIENT_POINT = "//div[@class='data-header__club-info']//following::span[5]//text()//following::span[1]//following::span[1]/text()[1]"
+        NUMBER_OF_TEAMS = "//div[@class='data-header__info-box']//div[@class='data-header__details']//li[1]/span/text()"
+        NUMBER_OF_PLAYERS = "//div[@class='data-header__info-box']//div[@class='data-header__details']//li[2]/span/text()"
```

Ruff

Line too long (122 > 120)

(E501)